<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecpeDetails</class>
 <widget class="QDialog" name="RecpeDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(117, 157, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="4">
      <widget class="QLabel" name="totalTimeLabel">
       <property name="text">
        <string>Total time</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="servingsLabel">
       <property name="text">
        <string>Servings</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ratingLabel">
       <property name="text">
        <string>Rating</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="cookTimeLabel">
       <property name="text">
        <string>Cook time</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="prepTimeLabel">
       <property name="text">
        <string>Prep. time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="ratingLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="servingsLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="cookTimeLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="prepTimeLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="totalTimeLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="ingredientsLabel">
       <property name="text">
        <string>Ingredients</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="directionLabel">
       <property name="text">
        <string>Direction</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="nutritionsLabel">
       <property name="text">
        <string>Nutritions</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="ingredientsText">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="nutritionsText">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPlainTextEdit" name="directionText">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="recipeNameLabel">
       <property name="text">
        <string>Recipe Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="recipeNameLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(76, 102, 255)</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RecpeDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RecpeDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
